code/__DEFINES/ai/ai.dm

```diff
@@ -76,6 +76,8 @@
 
 //Generic subtree defines
 
+/// default search range (tiles, passed to oview) when using find_and_set
+#define SEARCH_TACTIC_DEFAULT_RANGE 3
```

Comment:
I suspect what we were seeing before was world.view being used (IE 7 tiles) though I'd have to check what null does to say for sure.

Reply:
that's a fair point, if that's the default I'll change it to that. 3 tiles seemed a bit small with some of the behaviours involved